org-cyf-itd/content/blocks/step-3/instructions/index.md

```diff
@@ -15,6 +15,8 @@ Learn HTML and CSS by completing Unit 2 of [Khan Academy](https://www.khanacadem
 
 1. Create an [account on Khan Academy](https://www.khanacademy.org/computing/computer-programming/html-css)
 1. Go to the course: [Computer Programming: Unit 2](https://www.khanacademy.org/computing/computer-programming/html-css) HTML/CSS Making webpages
+1. Go to the course: [Computer Programming: Unit 2](https://www.khanacademy.org/computing/computer-programming/html-css) HTML/CSS Making webpages
```

Comment:
This looks like a duplicate of the previous bullet point? Is there a reason to add it twice?

```suggestion
```